qemu/qemu.go

```diff
@@ -1221,9 +1225,18 @@ func (config *Config) appendCPUs() {
 			SMPParams = append(SMPParams, fmt.Sprintf(",sockets=%d", config.SMP.Sockets))
 		}
 
+		if config.SMP.MaxCPUs > 0 {
+			if config.SMP.MaxCPUs < config.SMP.CPUs {
+				return fmt.Errorf("MaxCPUs must be equal to or greater than CPUs")
```

Comment:
Might be nice to include the actual values in the error message.

Reply:
yep, fixing